docs/en/observability/apm/known-issues.asciidoc

```diff
@@ -84,63 +85,26 @@ stream lifecycle configuration, such existing data streams become unmanaged for
 lifecycle configurations.
 
 // How to fix it
-Upgrading to 8.15.1 should fix any new indices created for the data stream. However,
-indices created in version 8.15.0 would remain unmanaged if the default ILM policy is
-used. One of the following approaches can be adopted to fix the unmanaged indices:
-
-. Manually delete the indices when they are no longer needed.
-. Explicitly configure APM data streams with the default data stream lifecycle config.
-Using this approach would migrate all data streams to use data stream lifecycles,
-which should be equivalent to the default ILM policies:
+Upgrading to 8.15.1 resolves the lifecycle issue for any new indices created for APM data streams.
+However, indices created in version 8.15.0 will remain unmanaged if the default ILM policy is in place.
+To fix these unmanaged indices, consider one of the following approaches:
+
+. Manually delete the unmanaged indices when they are no longer needed.
+. Explicitly configure APM data streams to use the default data stream lifecycle configuration.
+This approach migrates all affected data streams to use data stream lifecycles,
+maintaining behavior equivalent to the default ILM policies.
+Apply this fix only to data streams that are currently unmanaged due to default ILM policies.
```

Comment:
```suggestion
Apply this fix only to data streams that have unmanaged indices due to missing default ILM policies.
```

The datastream won't be unmanaged currently so a slight change of wordings.

Reply:
Ah! Good catch.